bookserver/crud.py

```diff
@@ -276,3 +278,39 @@ async def create_initial_courses_users():
                 course_id=12,
             )
         )
+
+
+async def update_user_state(user_data: schemas.LastPageData):
+    stmt = (
+        update(UserState)
+        .where(
+            (UserState.user_id == user_data.user.id)
```

Comment:
I don't understand this code -- where is `user` coming from in `user_data`?

Reply:
I think you just found a bug.